tutorials/userns.md

```diff
@@ -0,0 +1,65 @@
+# User namespaces and CRI-O
+
+The purpose of this document is to show how to configure user namespaces in CRI-O, as well as some of the options CRI-O supports for configuring user namespaces.
+
+## Setup
+
+### /etc/sub{g,u}id
+To start, the host will have to have `/etc/subuid` and `/etc/subgid` files set correctly. By default, the [library CRI-O uses for container storage](https://github.com/containers/storage) assumes there will be entries in each of these files for the `containers` user. If one would like to have a different user's entries in `/etc/sub?id` files, then the field `remap-user` and `remap-group` can be configured in `/etc/containers/storage.conf` in the `[storage.options]` table.
+
+Let's assume we want the IDs of the users and groups to begin on the host at 100000, and be each given ranges of 65536. For most containers, this will be more than enough. The contents of both `/etc/subuid` and `/etc/subgid` should be:
+```
+containers:100000:65536
+```
+
+### CRI-O configuration
+To enable pods to be able to use the userns-mode annotation, the pod must be allowed to interpret the experimental annotation `io.kubernetes.cri-o.userns-mode`.
+#### 1.23.0 and beyond
+In CRI-O versions greater than 1.23.0, this can be done by creating a custom workload. This can be done by creating a file with the following contents in /etc/crio/crio.conf.d/01-userns-workload.conf
```

Comment:
```suggestion
In CRI-O versions greater than 1.23.0, create a file with the following contents in /etc/crio/crio.conf.d/01-userns-workload.conf
```
"This can be done" got a little redundant.